### ISSUE TITLE
Feature Request: Alternative date formats

### ISSUE THREAD

**Issue by darrylgibbs** [emavgl/oinkoin#131]
Hi, firstly thanka for the amazing app!

Would it be possible to add the non-USA for lmat of DD-MM-YYYY or YYYY-MM-DD? asbbest as i can tell the US format is the only option, and as a non American i find this a tad confusing.

Thanks again!

**Comment by emavgl** [comment]
Hi! The date format depends on the language of the app. If you choice UK English, you should have the format you want.

The app tries to use the language of the phone and, if not supported, will use US locale.

I may try to separate the language and the date Format, however.

For understanding it better, which language does your phone have?

**Comment by darrylgibbs** [comment]
You're right. I hadn't force closed the app to make the change. All seems in order now.

My phone is set to English, but US English since I need the $ on my keyboard for the local currency (Brazil).

Thanks again. I really like this app!

**Comment by emavgl** [comment]
I see, thanks for explaining. Glad that the workaround works for you. I keep the PR open for adding an option in the preferences for changing the date format without changing the language. Have a nice day! :)
